agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -152,10 +156,16 @@
     }
 
     public static class Heartbeat {
-
         public long intervalSeconds = MINUTES.toSeconds(15);
     }
 
+    public static class Statsbeat {
+        public String ikey = "c4a29126-a7cb-47e5-b348-11414998b11e;IngestionEndpoint"; //workspace-aistatsbeat
+        public String endpoint = ConnectionString.Defaults.INGESTION_ENDPOINT; // this supports the government cloud
```

Comment:
I thought gov clouds required a different ingestion endpoint (at least those gov clouds that do not have access to the public internet)?

Reply:
I confirmed with Krishna, this end point will be used in gov clouds